lib/solveEquation/EquationOperations.js

```diff
@@ -220,12 +220,14 @@ function performTermOperationOnEquation(equation, op, term, changeType) {
 
 // Performs an operation of a term on an entire given expression
 function performTermOperationOnExpression(expression, op, term) {
-  const node = (Node.Type.isOperator(expression) ?
-    Node.Creator.parenthesis(expression) : expression);
-
+  if (Node.Type.isOperator(expression) &&
+    (op === '/' || op === '*' || op === '^')) {
+      var node = Node.Creator.parenthesis(expression);
```

Comment:
we only use `let` and `const`, so I think you'd need to do `let node;` before the if statement and then assign it inside the if/else (not super elegant, I know)